src/cascadia/UIHelpers/Converters.cpp

```diff
@@ -7,58 +7,84 @@
 
 namespace winrt::Microsoft::Terminal::UI::implementation
 {
-    winrt::hstring Converters::AppendPercentageSign(double value)
+    // Booleans
+    bool Converters::InvertBoolean(bool value)
     {
-        return to_hstring(gsl::narrow_cast<uint32_t>(std::lrint(value))) + L"%";
+        return !value;
     }
 
-    winrt::Windows::UI::Xaml::Media::SolidColorBrush Converters::ColorToBrush(const winrt::Windows::UI::Color& color)
+    winrt::Windows::UI::Xaml::Visibility Converters::InvertedBooleanToVisibility(bool value)
     {
-        return Windows::UI::Xaml::Media::SolidColorBrush(color);
+        return value ? winrt::Windows::UI::Xaml::Visibility::Collapsed : winrt::Windows::UI::Xaml::Visibility::Visible;
     }
 
-    winrt::Windows::UI::Text::FontWeight Converters::DoubleToFontWeight(double value)
+    // Numbers
+    double Converters::PercentageToPercentageValue(double value)
```

Comment:
Won't this cause the XAML UI to occasionally display something stupid like `99.000000000000000000000000001%`?

Reply:
making the signature `double (double)`, that isz

Reply:
That's a good point. I'll test this. The only time this should occur is when we convert between float and double, so I'll just find and test all the places where we do that (shouldn't be too many?).

Reply:
This does result in something stupid like that, but not in the way you thought: It's actually `AppendPercentageSign` which does that and we simply didn't notice because we were lucky.